src/yetibot/core/commands/karma.clj

```diff
@@ -1,60 +1,61 @@
 (ns yetibot.core.commands.karma
   (:require
+   [schema.core :as sch]
+   [yetibot.core.config :refer [get-config]]
    [yetibot.core.hooks :refer [cmd-hook]]
    [yetibot.core.models.karma :as model]
    [yetibot.core.commands.karma.specs :as karma.spec]
    [clojure.string :as str]
-   [clj-time.format :as fmt]
+   [clj-time.format :as t]
    [clojure.spec.alpha :as s]))
 
+(def config (:value (get-config sch/Any [:karma])))
+
+(def pos-emoji (or (-> config :emoji :positive) "🌈"))
+(def neg-emoji (or (-> config :emoji :negative) "⛈"))
```

Comment:
I wonder if/how we could expose the react-based method to the user in docs. Maybe a good candidate for more general docs once we revamp `help`. 🤔